### PR TITLE
[13.0][ADD] Add partner_sale_address module.

### PR DESCRIPTION
**Pull request by acsonefho**
Module used to restrict invoice/shipping addresses available on the sale order depending on the picked customer. Addresses available are contacts of the customer (or itself).

As the impact is only visual, no unit test needed.

Please tell me if I add this module into the correct repo.

Thanks for reviewers :)

### CODE REVIEW

**Comment by acsonefho** [comment]
Build failed but no errors related to this module.
`13.0` is already red.

File: partner_sale_address/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    partner_invoice_id = fields.Many2one(
+        domain="['|', ('parent_id', '=', partner_id), ('id', '=', partner_id)]",
```
Comment (ThomasBinsfeld) [inline partner_sale_address/models/sale_order.py]:
```suggestion
        domain="[('id', 'child_of', partner_id)]",
```

File: partner_sale_address/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    )
+    partner_shipping_id = fields.Many2one(
+        domain="['|', ('parent_id', '=', partner_id), ('id', '=', partner_id)]",
```
Comment (ThomasBinsfeld) [inline partner_sale_address/models/sale_order.py]:
```suggestion
        domain="[('id', 'child_of', partner_id)]",
```

File: partner_sale_address/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+This module restrict invoice and shipping addresses visible on the sale order depending on
```
Comment (ThomasBinsfeld) [inline partner_sale_address/readme/DESCRIPTION.rst]:
```suggestion
This module restricts invoice and shipping addresses visible on the sale order depending on
```

File: partner_sale_address/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2020-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner sale address",
+    "version": "13.0.1.0.0",
```
Comment (ThomasBinsfeld) [inline partner_sale_address/__manifest__.py:5]:
```suggestion
    "version": "13.0.1.0.0",
    "development_status": "Beta",
```

**Review by ThomasBinsfeld — CHANGES_REQUESTED** [review]

**Comment by ThomasBinsfeld** [comment]
I think this should be in sale-workflow, don't you?

**Comment by acsonefho** [comment]
> I think this should be in sale-workflow, don't you?

I put this module here because I think it's more about the contact part than the workflow. It doesn't block or update anything about the workflow.

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by chrisdec — APPROVED** [review]
Functional test

**Comment by dreispt** [comment]
Tests are failing. Restarted the build just in case.

File: partner_sale_address/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2020-Today: Odoo Community Association (OCA)
```
Comment (NL66278) [inline partner_sale_address/__init__.py:1]: There should be no copyright lines in init files and all license lines (in all files) should be https:

File: partner_sale_address/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2020-Today: Odoo Community Association (OCA)
```
Comment (NL66278) [inline partner_sale_address/__manifest__.py:1]: And do not put -Today in the copyright line. The last year, should be the year that the latest changes were made.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Nice addition, but needs some changes to comply with conventions.

**Comment by NL66278** [comment]
Questions and remarks not taken up

**Pull request closed without merging**